crates/runtime/tests/integration.rs

```diff
@@ -51,6 +51,7 @@ mod flight;
 mod github;
 mod graphql;
 mod iceberg;
+mod iceberg_catalog;
```

Comment:
This can be in the same module as the existing `iceberg` tests. If you want to divide them you can create a sub-module - but don't create an entirely new top-level module